openfl-tutorials/Federated_FedProx_PyTorch_MNIST_Workflow_Tutorial.ipynb

```diff
@@ -0,0 +1,343 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Federated FedProx PyTorch MNIST Workflow API Tutorial\n",
+    "This notebook mimics Federated_FedProx_PyTorch_MNIST_Tutorial using the workflow API.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Import the relevant libraries, choose an aggregation algorithm an optimizer and a loss function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "import torch.nn.functional as F\n",
+    "\n",
+    "import torch.utils\n",
+    "import torch.utils.data\n",
+    "import torchvision\n",
+    "import torchvision.transforms as transforms\n",
+    "\n",
+    "from openfl.utilities.optimizers.torch.fedprox import FedProxAdam\n",
+    "\n",
+    "from openfl.experimental.interface import FLSpec, Aggregator, Collaborator\n",
+    "from openfl.experimental.runtime import LocalRuntime\n",
+    "from openfl.experimental.placement import aggregator, collaborator\n",
+    "\n",
```

Comment:
Suggest splitting the cell here. Helps partition the context. The functions in this cell should be close to where they are actually called.

Reply:
done